ppr-ui/src/composables/useInputRules.ts

```diff
@@ -55,13 +55,23 @@ export const useInputRules = () => {
   }
 
   const isEmail = (): Array<Function> => [
-    (v: string) => !!v || 'Email address is required',
+    (v: string) => !!v || 'Enter an email address',
     (v: string) => {
       const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-      return pattern.test(v) || 'Valid email is required'
+      return pattern.test(v) || 'Enter a valid email address'
     }
   ]
 
+  const isPhone = (
+    maxDigits: number = null,
+    customMsg: string = null
+  ): Array<Function> => {
+    return [
+      v => ((v && maxDigits) ? v.length >= maxDigits : true) || 'Minimum 10 characters',
+      v => (v ? /^\d+$/g.test(v.replace('(', '').replace(') ', '').replace('-', '')) : true) || `${customMsg || 'Must contain numbers only'}`
```

Comment:
Could you elaborate on this regex? 
By looking at the message, i would assume it's checking for numbers only but it seems like it's doing a lot more.

Reply:
Very good question... v-mask includes the masked characters in the field.  We require only 10 digits.. The validation ensures that we are validating  for numbers not the extra characters.  The extra characters will not be stored in db...